controllers/server-controllers.js

```diff
@@ -44,12 +44,29 @@ exports.getCommentsByArticleId = (req, res, next) => {
 
 
 exports.getArticles= (req, res, next) => {
-    selectArticles()
-    .then((articles)=>{
-        res.status(200).send({articles});
-    })
-    .catch(next)
-}
+        const {topic}=req.query
+        if (topic)
+        {
+            selectArticles(topic)
+            .then((articles)=>{
+                res.status(200).send({articles});
+            })
+            .catch(next)
+        }
+        else
+        {
+            selectArticles()
+            .then((articles)=>{
+               res.status(200).send({articles});
+           })
+           .catch(next)
+        }
```

Comment:
Not sure you need this if statement, seem at though they both call the same function